src/components/send/sendViews/initial.view.tsx

```diff
@@ -1261,6 +1301,9 @@ export function SendInitialView({ onNextScreen, setClaimLink, setTxHash, setChai
                                                               src={token.logo}
                                                               className="h-6 bg-white"
                                                               loading="eager"
+                                                              onError={(e: any) => {
```

Comment:
What's the purpose of dis?

Reply:
Ipfs sometimes ratelimits, this handles the error silently